src/bot.ts

```diff
@@ -0,0 +1,58 @@
+import { Client, Constructable, Message } from 'discord.js';
+import { Settings } from './services/settings';
+import { AvailableSettings } from './enums/available-settings';
+import { Output } from './services/output';
+import { ICommand } from './interfaces/command';
+import { Licence } from './commands/licence';
+import { Ping } from './commands/ping';
+
+export class Bot {
+    private client = new Client();
+    private commands?: Record<string, Constructable<ICommand>>;
+
+    public async liftOff(): Promise<void> {
+        await this.login();
+
+        this.client.on('ready', () => {
+            Output.line(`Logged in as ${this.client.user?.tag}`);
```

Comment:
waarin gebruik je hier backticks ipv normale aanhalingstekens?

Reply:
Dat is een template literal, zo kan ik die `this.client.user?.tag` in de string pompen zonder concatten

Reply:
oh cool